MapboxNavigation/CarPlayNavigationViewController.swift

```diff
@@ -175,12 +179,33 @@ public class CarPlayNavigationViewController: UIViewController {
         super.viewSafeAreaInsetsDidChange()
         
         // Adjust the map’s vanishing point to counterbalance the side maneuver panels by extending the view off beyond the other side of the screen.
-        if let mapView = mapView {
-            mapViewRightSafeAreaBalancingConstraint?.constant = -mapView.safeArea.right
-            mapViewLeftSafeAreaBalancingConstraint?.constant = mapView.safeArea.left
+        guard let mapView = mapView else { return }
+        
+        mapViewRightSafeAreaBalancingConstraint?.constant = -mapView.safeArea.right
+        mapViewLeftSafeAreaBalancingConstraint?.constant = mapView.safeArea.left
+        view.setNeedsUpdateConstraints()
+        
+        mapView.enableFrameByFrameCourseViewTracking(for: 1)
+        
+        if !tracksUserCourse {
+            mapView.setContentInset(overviewContentInsets(), animated: false)
+            mapView.fit(to: navigationService.route, facing: 0, padding: .zero, animated: false)
```

Comment:
Now if the user pans, disabling course tracking but without entering overview mode, the map will revert to course tracking once the navigation bar goes away. I guess that’s OK but somewhat unexpected.

Reply:
You're right. We probably need a flag `isOverviewing` here similar to RouteMapViewController.
Panning around is currently not possible either with course tracking on/off nor overviewing (navigation bar popping in triggers viewSafeAreaInsetsDidChange() which in turn, fits the camera to the route).

Reply:
@JThramer pointed out that this is actually pretty fortuitous. There isn’t a Resume or Recenter button like in NavigationViewController, and there’s no location tracking button like during the CarPlay browsing activity, so this code path is actually the most convenient way to get back into course tracking mode. Otherwise, the user has to tap the Overview button and tap the location tracking button after that.

Reply:
Disagree. This behavior is actually better than it was previously, as there is no resume button in CarPlay.